### PR TITLE
simple hello world structure for translating transitions

### PR DESCRIPTION
**Pull request by Chestshire**
- Added a template file only for testing
- Modified the template files for transitions

Results:
- could print the function declarations for existing transitions (using template files)
- could identify whether a transition has a keyword expression or not (e.g., do, when, ...)
- generate some dummy placeholders for template values

### CODE REVIEW

File: org.alloytools.alloy.application/src/main/java/templates/TestTemplateFile.vm [hunk 1]
```diff
@@ -0,0 +1,54 @@
+# --- Signatures ---
```
Comment (RuomeiYan) [inline org.alloytools.alloy.application/src/main/java/templates/TestTemplateFile.vm:1]: Do you want to merge this test template file?

Comment (Chestshire) [reply org.alloytools.alloy.application/src/main/java/templates/TestTemplateFile.vm:1]: sure

File: org.alloytools.alloy.application/src/main/java/templates/VelocityTemplateFile.vm [hunk 2]
```diff
@@ -1,6 +1,80 @@
+# TODO: add some comments?
```
Comment (RuomeiYan) [inline org.alloytools.alloy.application/src/main/java/templates/VelocityTemplateFile.vm:1]: Do we need to put TODO's like this into the code? If there's a bug or performance bottleneck we need to check out later, I am fine with leaving a TODO comment there. I feel like a comment saying to add more comments is not necessary.

File: org.alloytools.alloy.application/src/main/java/templates/TestTemplateFile.vm (hunk 1 above)
Comment (RuomeiYan) [reply org.alloytools.alloy.application/src/main/java/templates/TestTemplateFile.vm:1]: Why do we need it? I feel like we should keep our code repository clean and concise.

**Review by RuomeiYan — APPROVED** [review]
A lot of code that is still under development, like placeholder and TODO comments, is also committed. I think the best practice would be to separate them into a development branch and don't commit them into our master branch.
However, I agree that we can merge this PR for now and clean things up later.

**Pull request merged**